src/utils/is-error.ts

```diff
@@ -0,0 +1,11 @@
+// We allow some additional attached properties for Errors
+interface NftError {
+  code?: string | number
+  message?: string
+}
+
+export default function isError(err: unknown): err is NftError {
+  return (
+    typeof err === 'object' && err !== null && ('code' in err || 'message' in err)
```

Comment:
Isn’t type of Error “function”?  

CC @mrmckeb for helpers

Reply:
I copied/modified this helper from the Next.js suite here https://github.com/vercel/next.js/blob/f8e283c7ba9f8f684511c00aee5e6742123dd2ba/packages/next/lib/is-error.ts

Reply:
Oh my bad, I tried `console.log(typeof Error)` but that should have been `console.log(typeof new Error())` which is the expected function.

That said, we should probably use the helpers that @mrmckeb wrote for the api.

Reply:
Let's revisit this when we update `typescript` in this repo, that can be another PR

Reply:
Updated to remove these type checks since they aren't actually needed yet as discussed